### ISSUE TITLE
Make it possible to stream EuXFEL data on disk

### ISSUE THREAD

**Issue by FilipeMaia** [FXIhub/hummingbird#133]
Using [EXtra-dat](https://github.com/European-XFEL/EXtra-dat).

**Comment by andyofmelbourne** [comment]
Are you not already able to stream data using karabo-bridge-serve-files, or is the aim here to stream _calibrated_ data on disk?

P.S. I am struggling to find any information about how to apply facility calibrations to the data (streamed or not) at the [euxfel](https://rtd.xfel.eu/docs/data-analysis-user-documentation/en/latest/), despite a few references to the fact that this is now possible.

**Comment by FilipeMaia** [comment]
Yes you can use an external tool to stream the files. The idea was to have it builtin like it happens at LCLS.
But your question regarding calibrated data is important. I don't know how to do it and it would be quite useful.
